pkg/platform/packet/packet.go

```diff
@@ -34,20 +36,22 @@ import (
 )
 
 type workerPool struct {
-	Name           string `hcl:"pool_name,label"`
-	Count          int    `hcl:"count"`
-	DisableBGP     bool   `hcl:"disable_bgp,optional"`
-	IPXEScriptURL  string `hcl:"ipxe_script_url,optional"`
-	OSArch         string `hcl:"os_arch,optional"`
-	OSChannel      string `hcl:"os_channel,optional"`
-	OSVersion      string `hcl:"os_version,optional"`
-	NodeType       string `hcl:"node_type,optional"`
-	Labels         string `hcl:"labels,optional"`
-	Taints         string `hcl:"taints,optional"`
-	SetupRaid      bool   `hcl:"setup_raid,optional"`
-	SetupRaidHDD   bool   `hcl:"setup_raid_hdd,optional"`
-	SetupRaidSSD   bool   `hcl:"setup_raid_ssd,optional"`
-	SetupRaidSSDFS bool   `hcl:"setup_raid_ssd_fs,optional"`
+	Name                  string            `hcl:"pool_name,label"`
+	Count                 int               `hcl:"count"`
+	DisableBGP            bool              `hcl:"disable_bgp,optional"`
+	IPXEScriptURL         string            `hcl:"ipxe_script_url,optional"`
+	OSArch                string            `hcl:"os_arch,optional"`
+	OSChannel             string            `hcl:"os_channel,optional"`
+	OSVersion             string            `hcl:"os_version,optional"`
+	NodeType              string            `hcl:"node_type,optional"`
+	Labels                string            `hcl:"labels,optional"`
+	Taints                string            `hcl:"taints,optional"`
+	ReservationIDs        map[string]string `hcl:"reservation_ids,optional"`
+	ReservationIDsDefault string            `hcl:"reservation_ids_default,optional"`
```

Comment:
Maybe `DefaultReservationID` ? It sounds better to me.

Reply:
Hm, I don't think it's worth diverging from terraform. For users coming from there, this would look simpler, and if we really want to, we can rename later (like deprecate, etc.)?

Reply:
Well, I'm not sure if we should care so much about the Terraform here. `reservation_ids_default` suggests, that this takes multiple values, but I'm fine with it as it is. Feel free to mark as resolved if you think that's better for now.

Reply:
Ok, thanks!